deltachat-ios/AppDelegate.swift

```diff
@@ -428,6 +433,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         // if at some point we do per-message-push-notifications, we need to tweak this gate.
         let nowTimestamp = Double(Date().timeIntervalSince1970)
         if nowTimestamp < bgIoTimestamp + 60 {
+            self.pushToDebugArray("e2")
             logger.info("➡️ fetch was just executed, skipping")
             completionHandler(.newData)
```

Comment:
we are skipping calls far too often, the condition seems to be wrong:

<img width=320 src=https://user-images.githubusercontent.com/9800740/163693335-c0bae17c-af05-4802-a874-132f0b854606.PNG>

does not happen always, screenshot is from iphone7, the other testphones do not have that issue yet.  once we are in this state, however, we're kind of stuck.

Reply:
`bgIoTimstamp` must probably not be set on init, see comment above.

Reply:
not sure at the end as also `didFinishLaunchingWithOptions` starts IO, however, maybe does not run long enough. we need more information ...